packages/backend/src/models/SavedSemanticViewerChartModel.ts

```diff
@@ -299,23 +300,6 @@ export class SavedSemanticViewerChartModel {
         return savedSemanticViewerChartVersionUuid;
     }
 
-    static async generateSavedSemanticViewerChartSlug(trx: Knex, name: string) {
```

Comment:
MOve this static method to a new method in utils, to be reused in different models